plugin-jdbc-duckdb/src/main/java/io/kestra/plugin/jdbc/duckdb/Query.java

```diff
@@ -77,24 +77,30 @@ public class Query extends AbstractJdbcQuery implements RunnableTask<Query.Outpu
     @PluginProperty(dynamic = false)
     protected List<String> outputFiles;
 
+    private Path databaseFile;
```

Comment:
You should add a `@Getter(AccessLevel.NONE)` and a transient, if not the properties will be public

Reply:
This was existing in the Query task, I'll update both places.

Reply:
done